src/stdlib/lv_string.h

```diff
@@ -48,6 +48,15 @@ void * lv_memcpy(void * dst, const void * src, size_t len);
  */
 void lv_memset(void * dst, uint8_t v, size_t len);
 
+/**
+ * @brief Move a block of memory from source to destination
+ * @param dst Pointer to the destination array where the content is to be copied.
+ * @param len Pointer to the source of data to be copied.
+ * @param count Number of bytes to copy.
+ * @return Pointer to the destination array.
+ */
```

Comment:
/**

@brief Move a block of memory from source to destination
@param dst Pointer to the destination array where the content is to be copied.
@param src Pointer to the source of data to be copied.
@param len Number of bytes to copy.
@return Pointer to the destination array.
*/